src/modules/form/components/group/Signature.tsx

```diff
@@ -21,25 +27,25 @@ const Signature = ({ item, renderChildItem }: GroupItemComponentProps) => {
       }}
     >
       <fieldset style={{ border: 'none', margin: 'none', padding: 'none' }}>
-        {item.text && (
-          <legend>
-            <Typography
-              variant='h5'
-              color={theme.palette.primary.main}
-              sx={{ mb: 2 }}
-            >
-              {item.text}
+        <Stack gap={2}>
+          {item.text && (
+            <legend>
+              <Typography variant='h5' color={theme.palette.primary.main}>
+                {item.text}
+              </Typography>
+            </legend>
+          )}
+          <Grid container direction='row' columnGap={viewOnly ? 6 : 2}>
+            {renderChildItem &&
+              item.item?.map((childItem) => renderChildItem(childItem))}
+          </Grid>
+          {!viewOnly && (
+            <Typography variant='body2'>
+              By signing, I do hereby certify that the above information is
+              true, accurate and complete to the best of my knowledge
```

Comment:
This is a bit of scope creep but I think this language isn't quite right. Folks aren't always signing in this context. They are also signing to allow release of information. Maybe:
"Entering my name and selecting this box constitutes my electronic signature and agreement to be legally bound"